VultisigApp/VultisigApp/View Models/EncryptedBackupViewModel.swift

```diff
@@ -329,7 +329,14 @@ class EncryptedBackupViewModel: ObservableObject {
                     return
                 }
                 
-                importedFileName = url.lastPathComponent
+                if url.pathExtension.lowercased() == "txt" {
```

Comment:
I think you don't need to do this check , you can just do 
`importedFileName = url.lastPathComponent.replacingOccurrences(of: ".txt", with: ".vult")`

Reply:
fixed